charts/jaeger/values.yaml

```diff
@@ -163,10 +163,14 @@ ingester:
   extraSecretMounts: []
   extraConfigmapMounts: []
 
+  serviceMonitor:
+    enabled: false
+    additionalLabels: {}
+
 agent:
   podSecurityContext: {}
   securityContext: {}
-  enabled: true
+  enabled: false
```

Comment:
This should not be changed, the default is to deploy an agent daemonset and is unrelated to security monitor.